pkg/daemon/containerrecreate/crr_daemon_util.go

```diff
@@ -114,7 +118,11 @@ func getCurrentCRRContainersRecreateStates(
 				syncContainerStatus.Ready {
 				currentState.Phase = appsv1alpha1.ContainerRecreateRequestSucceeded
 			}
-
+			if crr.Spec.Strategy.ForceRecreate && (len(forceKillContainerStatuses) == 0 ||
```

Comment:
len(forceKillContainerStatuses) checking is not necessary, containerIsNotForceKilled will check whether recreateState == nil , plz remove len(forceKillContainerStatuses) checking